sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -738,9 +756,20 @@ struct _pi_queue : _pi_object {
                                bool OKToBatchCommand = false);
 
   // If there is an open command list associated with this queue,
-  // close it, execute it, and reset ZeOpenCommandList, ZeCommandListFence,
-  // and ZeOpenCommandListSize.
-  pi_result executeOpenCommandList();
+  // close it, execute it, and reset the corresponding OpenCommandList.
+  // If IsCopy is 'true', then the OpenCommandList containing copy commands is
+  // executed. Otherwise OpenCommandList containing compute commands is
+  // executed.
+  pi_result executeOpenCommandList(bool IsCopy);
+
+  // Wrapper function to execute both OpenCommandLists (Copy and Compute).
+  pi_result executeOpenCommandLists() {
```

Comment:
To make it more distinct from the `executeOpenCommandList`:
```suggestion
  pi_result executeAllOpenCommandLists() {
```

Reply:
Also please add comment about how one should choose between the 2 versions. I.e. in what circumstances both batches need to be closed.

Reply:
Done. Thanks.